imagepicker/src/main/java/com/esafirm/imagepicker/helper/UiUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.esafirm.imagepicker.helper;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.Context;
+import android.content.res.Configuration;
+import android.os.Build;
+import android.view.Window;
+import android.view.WindowManager;
+
+public class UiUtil {
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+    public static void setStatusBarColor(Activity activity, int color) {
+        Window window = activity.getWindow();
+
+        window.clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
+
+        window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
+
+        window.setStatusBarColor(color);
+    }
+
+    public static boolean isLandscape(Context context) {
```

Comment:
Is this used in the codebase? 
I believe in [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) 😸